src/components/MyAccount/Settings/AccountSettingsDisplayOptions.tsx

```diff
@@ -29,7 +30,50 @@ export const AccountSettingsDisplay = ({ patron }: { patron: Patron }) => {
   return <>{terms.map(buildListElementsWithIcons)}</>
 }
 
-export const AccountSettingsForm = ({ patron }: { patron: Patron }) => {
+export const AccountSettingsForm = ({
+  patron,
+  setIsFormValid,
+}: {
+  patron: Patron
+  setIsFormValid: Dispatch<React.SetStateAction<boolean>>
+}) => {
+  const [formData, setFormData] = useState({
+    primaryPhone: patron.phones[0]?.number,
+    email: patron.emails[0],
+  })
+
+  const isFormValid = useCallback(() => {
+    const phoneRegex = /^[0-9-]+$/
```

Comment:
hm okay. how about we strip out non number characters and verify that the leftover string is 10 digits.

Reply:
alternately, you can use this regex `(\d{3})[^0-9]{1,2}?(\d{3})[^0-9]?(\d{4})` which matches these variations:
123-456-7890
1234567890
(123) 456-7890
1 (234) 567 8900

Reply:
Actually I think the move is to have place holder text like this:
`000 000 0000` 
And then verify that someone has submitted 10 digits minus whitespace.